src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -78,4 +92,73 @@ public ResponseEntity<Object> getIssuerConfig(@PathVariable("issuer-id") String
         return ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
     }
 
+    @GetMapping("/{issuer-id}/credentials-supported")
+    public ResponseEntity<Object> getCredentialsSupportedForIssuer(@PathVariable("issuer-id") String issuerId,
```

Comment:
We can use method name as `getCredentialsSupported`.

Reply:
Api url should be `/{issuer-id}/credentialsSupported`

Reply:
changed